invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/state/actions.js

```diff
@@ -1,13 +1,10 @@
-import {
-  IS_LOADING,
-  SUCCESS,
-  HAS_ERROR,
-  ACTION_IS_LOADING,
-  ACTION_SUCCESS,
```

Comment:
`fetchLoanDetails` and `performLoanAction` are using the same types?

Reply:
We removed the explicit action types and delegated everything to loanDetails as you spotted. We believe there is no particular reason for having them since the errors will point to the actual problem that was raised. Also the UI doesn't break with a spinner in the middle of LoanDetails. Please feel free to express your concerns!